tests/recorder.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+/*
+    Usage:
+
+        node recorder --test={#} --description="This test describes {xyz} conditions"
+
+    Requirements:
+
+        -  The `test{#}` directory must exist within the `fixtures` directory
+        -  The `test{#}` directory must contain `payload.json` and `headers.json` files, representing info POSTed form GitHub
+
+*/
```

Comment:
Awesome! Thanks for documenting this. Another nice addition might be an explanation of what `recorder.js` can be used for, either here or in one of our Markdown files or both. That might help people get up-and-running with the recorder.